README.md

```diff
@@ -39,21 +50,40 @@ See aces/Loris README.md for further information and Loris installation informat
 
  * What is the database name? $dbname
  * What is the database host? $dbhost
- * What is the MySQL user? $lorisuser 
+ * What is the MySQL user? $lorisuser [Use the same mysql user from the Loris installation, i.e. _lorisuser_]
  * What is the MySQL password? 
  * What is the Linux user which the installation will be based on? $lorisadmin
  * What is the project name? $projectname
  * What is your email address? 
  * What prod file name would you like to use? default: prod  [leave blank]
  * Enter the list of Site names (space separated) site1 site2
 
-Ensure that /home/$lorisadmin/.bashrc includes the statements: 
+  If the imaging install script reports errors in creating directories (due to /data/ mount permissions), manually execute mkdir and chmod commands starting at [imaging_install.sh:L90](https://github.com/aces/Loris-MRI/blob/16.0-dev/imaging_install.sh#L90)
+
+  The installer will make apache (www-data) part of the lorisadmin linux group.  
+
```

Comment:
@christinerogers,

Hi Christine,
so this now needs to be updated, as we are now changing the group to be apache (www-data) and then using the chmod g+s;
https://github.com/aces/Loris-MRI/pull/129